### PR TITLE
Add Candy Kata to README section. closes ticket #133

### PR DESCRIPTION
**Pull request by mvdietz**
Signed-off-by: Matthew Dietz <Matthew.Dietz@bnymellon.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,7 +15,9 @@ bin
 .idea/workspace.xml
 .idea/libraries
 .idea/gradle.xml
+.idea/compiler.xml
```
Comment (donraab) [inline .gitignore:18]: Hi @mvdietz, please revert these changes as they should be in a separate PR.

File: README.md [hunk 2]
```diff
@@ -55,18 +55,20 @@ In the "Open Project" dialog you might select the option "Open Required Projects
...
+candy-kata/src/tests
 ```
 
 To get started, you can refer to slides for the [Instruction and Pet Kata](http://eclipse.github.io/eclipse-collections-kata/) to learn how to set-up Kata, basic features of Eclipse Collections corresponding to each Pet Kata exercise and then solutions. 
+Check out the [pet kata solutions module tests](https://github.com/eclipse/eclipse-collections-kata/tree/master/pet-kata-solutions/src/test/java/org/eclipse/collections/petkata) for your reference.
 
 To learn wider range of functionalities, slides for [Company Kata](http://eclipse.github.io/eclipse-collections-kata/company-kata) are now available online as well. 
 
-Check out the [pet kata solutions module tests](https://github.com/eclipse/eclipse-collections-kata/tree/master/pet-kata-solutions/src/test/java/org/eclipse/collections/petkata) for your reference.
+To learn more about the Bag data structure, take a look at the [Candy Kata](candy-kata)
```
Comment (donraab) [inline README.md:71]: Please add a period at the end of sentence to be consistent with the rest of the paragraphs.

**Comment by mvdietz** [comment]
I've made the changes you suggested Don. Please review.

**Comment by donraab** [comment]
LGTM. Thanks for the contribution @mvdietz !
